special/SheetList.php

```diff
@@ -87,7 +90,8 @@ public function execute($par) {
 		$table = "{| class=\"mw-datatable\" style=\"width:100%\"\n";
 		$msgModName = wfMessage('tilesheet-mod-name');
 		$msgSizesName = wfMessage('tilesheet-sizes');
-		$canEdit = in_array("edittilesheets", $this->getUser()->getRights());
+		$canEdit = in_array("edittilesheets", MediaWikiServices::getInstance()->getPermissionManager()
```

Comment:
not important: Could be changed to:
`$canEdit = MediaWikiServices::getInstance()->getPermissionManager()->userHasRight( $this->getUser(), 'edittilesheets' )`